### DISCUSSION TITLE
Rename ChainID to ?

### DISCUSSION THREAD

**Discussion by P1sar** [ChainSafe/chainbridge-core#82] [Ideas]
We really should start the discussion about renaming ChainID param in Chainbridge that conflicts with the EIP-155's ChainID and brings a lot of confusion. This param should be renamed in both Chainbridge and Solidity.  I this thread is great start if anyone has any ideas!

**Comment by P1sar** [comment]
mu suggestions are: bridgeID or domainID

**Reply by GregTheGreek** [reply]
BridgeId sounds more explicit but I'm indifferent.

**Comment by GregTheGreek** [comment]
We may also look to the [CAIP](https://github.com/ChainAgnostic/CAIPs) standards being led by members from walletconnect, metamask and others. 

If it makes sense we should arguably even participate in the discussions with them.

**Comment by ansermino** [comment]
I personally like the term `domainId`. This seems very appropriate and flexible for multiple chains (or even potentially non-blockchain entities in the future) and for multiple deployments on a single chain. 

(`bridgeId` I think would be too inflexible as it could be interpreted as defining a bridge, but really it's only defining a single point on a potentially multidirectional bridge)

**Comment by P1sar** [comment]
Ok, so we decide to use domainID instead of chainID for now
